components/script/dom/htmllinkelement.rs

```diff
@@ -472,6 +488,18 @@ impl HTMLLinkElement {
         );
     }
 
+    /// <https://html.spec.whatwg.org/multipage/#attr-link-disabled>
+    fn handle_stylesheet_disabled(&self, disabled: bool) {
```

Comment:
In this case, the naming makes the functionality of this method really ambiguous, so maybe go with `handle_stylesheet_disabled_attribute_changed`.

Reply:
Went with `handle_disabled_attribute_change` since the explicitly enabled flag needs to be set regardless of whether there is a stylesheet yet.